include/onnxruntime/core/framework/data_types.h

```diff
@@ -258,7 +258,7 @@ class TensorTypeBase : public DataTypeImpl {
 
   virtual MLDataType GetElementType() const {
     // should never reach here.
-    ONNXRUNTIME_NOT_IMPLEMENTED(__FUNCTION__, " is not implemented");
+    ORT_NOT_IMPLEMENTED(__FUNCTION__, " is not implemented");
```

Comment:
yeah, having some macros that have ONNXRUNTIME_ and some ORT_ doesn't look very clean.

Reply:
Ok, I'll make them all ORT_